### PR TITLE
File specific overwrote all other parameters to default

### PR DESCRIPTION
**Pull request by zrolfs**

### CODE REVIEW

**Comment by zrolfs** [comment]
There's a hefty unit test, because it seems like this is the third time I've had to fix a parameter overwrite issue...

File: EngineLayer/CommonParameters.cs [hunk 1]
```diff
@@ -104,15 +104,5 @@ public CommonParameters()
         public bool TrimMsMsPeaks { get; private set; }
         public bool UseDeltaScore { get; private set; }
         public bool CalculateEValue { get; private set; }
-
-        public CommonParameters Clone()
```
Comment (rmillikin) [inline EngineLayer/CommonParameters.cs]: plz put this back

File: TaskLayer/MetaMorpheusTask.cs [hunk 2]
```diff
@@ -181,7 +188,28 @@ public static CommonParameters SetAllFileSpecificCommonParams(CommonParameters c
                 zDotIons: zdotIons,
                 precursorMassTolerance: precursorMassTolerance,
                 productMassTolerance: productMassTolerance,
-                digestionParams: fileSpecificDigestionParams);
+                digestionParams: fileSpecificDigestionParams,
+
+                //NEED THESE OR THEY'LL BE OVERWRITTEN
```
Comment (rmillikin) [inline TaskLayer/MetaMorpheusTask.cs]: 👍

File: EngineLayer/CommonParameters.cs (hunk 1 above)
Comment (zrolfs) [reply EngineLayer/CommonParameters.cs]: Is there a reason? I didn't see it being used for anything.

Comment (rmillikin) [reply EngineLayer/CommonParameters.cs]: it's not used right now, but I'd like to use it to duplicate a task in the GUI

**Pull request merged**